graphs/ProjectRuntime.fbp

```diff
@@ -8,8 +8,9 @@ HoldConnection(ui/HoldRuntimeConnection) CONTEXT -> IN ConnectedContext(core/Mer
 'true' -> REQUIREGRAPHS HoldConnection
 
 # If not, the connect
-'runtime' -> KEY GetDefinition(objects/GetObjectKey)
+'runtime' -> STRING SendRuntimeKey(strings/SendString) OUT -> KEY GetDefinition(objects/GetObjectKey)
```

Comment:
Why do we need this now, and not before?

Reply:
The original, pre Process API noflo-objects treated the `KEY` port as essentially a control port. The new one treats it as regular input. This is great for getting rid of race conditions, but the downside is that graphs using the component(s) have to be adapted like we did here

Reply:
Ok